tests/nonroot_test.go

```diff
@@ -32,24 +32,15 @@ var _ = Describe("[sig-compute]NonRoot feature", func() {
 		tests.BeforeTestCleanup()
 	})
 
-	virtioFsVM := func() *v1.VirtualMachineInstance {
-		name := "test"
-		return tests.NewRandomVMIWithPVCFS(name)
-	}
-
-	table.DescribeTable("should cause fail in creating of vmi with", func(createVMI func() *v1.VirtualMachineInstance, neededFeature, feature string) {
-		if neededFeature != "" && !checks.HasFeature(neededFeature) {
-			Skip(fmt.Sprintf("Missing %s, enable %s featureGate.", neededFeature, neededFeature))
+	It("should cause fail in creating of vmi with [test_id:7127]VirtioFS", func() {
+		if !checks.HasFeature(virtconfig.VirtIOFSGate) {
+			Skip(fmt.Sprintln("Missing VirtioFS feature gate, enable ExperimentalVirtiofsSupport featureGate."))
 		}
 
-		vmi := createVMI()
+		vmi := libvmi.NewTestToolingFedora(libvmi.WithVirtioFS("something"))
 		_, err = virtClient.VirtualMachineInstance(util.NamespaceTestDefault).Create(vmi)
-		Expect(err).To(HaveOccurred())
-		Expect(err.Error()).To(And(ContainSubstring(feature), ContainSubstring("nonroot")))
-
-	},
-		table.Entry("[test_id:7127]VirtioFS", virtioFsVM, virtconfig.VirtIOFSGate, "VirtioFS"),
-	)
+		Expect(err.Error()).To(And(MatchError(ContainSubstring("VirtioFS")), MatchError(ContainSubstring("nonroot"))))
```

Comment:
I think you don't need to `Expect(err.Error())`; `Expect(err)...` would be enough.

Reply:
:+1: